python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/vxlan_interface.schema.yml

```
@@ -37,6 +37,12 @@ keys:
                 type: int
                 convert_types:
                   - str
+              ipv6_underlay:
+                type: dict
+                description: Enable bridging over EVPN IPv6 VXLAN underlay.
+                keys:
+                  enabled:
+                    type: bool
```

If there are no other "ipv6 underlay" commands under the vxlan interface, I think we should simplify this to:
```suggestion
              ipv6_underlay:
                type: bool
                description: Enable bridging over EVPN IPv6 VXLAN underlay.
```

---

If we keep it nested, please move the description to the enabled key.

---

updated model to `encapsulations`